### PR TITLE
Rewrite for paypal v2 orders API

### PR DESCRIPTION
**Pull request by Shin--**

### CODE REVIEW

File: django_paypal/models.py [hunk 1]
```diff
@@ -1,187 +1,158 @@
...
+class PaypalWebhook(models.Model):
+    webhook_id = models.CharField(_('Webhook ID'), max_length=255, unique=True)
+    url = models.CharField('URL', max_length=255, unique=True)
+    events = JSONField(_('Active Webhook Events'))
+
+
+# The following models solely exist to keep payments made in version <0.3.0 stored in the database.
+# They are not used in the current version of the app and have been created using /v1/payments.
+# They might get removed in a future version.
+class LegacyPaypalPayment(models.Model):
```
Comment (vincenz-e) [inline django_paypal/models.py:68]: Maybe add a deprecated decorator.
```
from warnings import deprecated

@deprecated("Deprecated model for deprecated PayPal API")
class LegacyPaypalPayment(models.Model): ...
```

File: django_paypal/models.py [hunk 2]
```diff
@@ -1,187 +1,158 @@
...
-    shipping_address_state = models.CharField(_("shipping address state"), max_length=100, blank=True)
-    shipping_address_phone = models.CharField(_("shipping address phone"), max_length=50)
-    shipping_address_normalization_status = models.CharField(_("shipping address normalization status"), max_length=50, default="UNKNOWN")  # UNKNOWN or UNNORMALIZED_USER_PREFERRED or NORMALIZED or UNNORMALIZED
-    shipping_address_type = models.CharField(_("shipping address type"), max_length=50, default="HOME_OR_WORK")  # HOME_OR_WORK or GIFT or other
-    shipping_address_recipient_name = models.CharField(_("shipping address recipient name"), max_length=127)
+        verbose_name = _('Legacy Paypal Payment (payments v1 API)')
+        verbose_name_plural = _('Legacy Paypal Payments (payments v1 API)')
+
+
+class LegacyPaypalTransaction(models.Model):
```
Comment (vincenz-e) [inline django_paypal/models.py:102]: Maybe also deprecated decorator

File: django_paypal/models.py [hunk 3]
```diff
@@ -1,187 +1,158 @@
...
-        verbose_name = _("Paypal Transaction")
-        verbose_name_plural = _("Paypal Transactions")
+        verbose_name = _('Legacy Paypal Transaction (payments v1 API)')
+        verbose_name_plural = _('Legacy Paypal Transactions (payments v1 API)')
 
 
-@python_2_unicode_compatible
-class PaypalItem(models.Model):
-    transaction = models.ForeignKey(PaypalTransaction, verbose_name=_("transaction"), related_name='items', on_delete=models.PROTECT)
+class LegacyPaypalItem(models.Model):
```
Comment (vincenz-e) [inline django_paypal/models.py:160]: Maybe also deprecated decorator

File: django_paypal/models.py (hunk 1 above)
Comment (Shin--) [reply django_paypal/models.py:68]: I would, if there would be a decorator for that in python standard libraries

**Review by vincenz-e — APPROVED** [review]

**Pull request merged**